hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/dsl/ResourceServerConfigurationCustomizerDsl.kt

```diff
@@ -56,7 +56,7 @@ class ResourceServerConfigurationCustomizer {
   lateinit var authorizeExchangeCustomizer: AuthorizeExchangeCustomizer
 
   companion object {
-    fun build(dsl: ResourceServerConfigurationCustomizerDsl.() -> Unit): ResourceServerConfigurationCustomizer =
+    operator fun invoke(dsl: ResourceServerConfigurationCustomizerDsl.() -> Unit): ResourceServerConfigurationCustomizer =
```

Comment:
I couldn't work out how to do this with invoke previously but the penny dropped after reading up on it.